misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/AnyExpectationsSpec.kt

```diff
@@ -53,6 +53,8 @@ abstract class AnyExpectationsSpec(
     toBeAnInstanceOfSuperType: Feature1<out Any?, Expect<SuperType>.() -> Unit, SuperType>,
     toBeAnInstanceOfSubTypeFeature: Feature0<out Any?, SubType>,
     toBeAnInstanceOfSubType: Feature1<out Any?, Expect<SubType>.() -> Unit, SubType>,
+    notToBeInstanceOfFeature:Feature0<out Any?, Boolean>,
+    notToBeAnInstanceOf: Feature1<out Any?, Expect<SuperType>.() -> Unit, Boolean>,
```

Comment:
please also add a test-case

Reply:
Here I runned into a problem, I can't actually find a way to test this out. Could you maybe hint me out how I should go about adding a unifySignatures that would work for feature0/1 with same return type?

Reply:
this function already exists actually and is already used by `toBeAnInstanceOfSubTypeFeature` and `toBeAnInstanceOfSubType` in AnyExpectationsSpec.

Following the things you should do:
- add notToBeInstanceOfFeature and notToBeAnInstanceOf to both includes of of the SubjectLessSpec
- add a `describeFun` (see line 683 toBeAnInstanceOfIntFeature) and add similar test cases